popup.js

```diff
@@ -348,23 +360,32 @@ function resetFont() {
     }
   }
 
-  // Serializes the function and sends to target page.
-  var code = PREVIEWER_resetFont.toString() + ' PREVIEWER_resetFont(' + stringifyArgs(fontOptionIds) + ');';
-  chrome.tabs.executeScript(null, {code: code});
+  // Execute PREVIEWER_resetFont on the target page
+  chrome.scripting.executeScript({
+    target: {tabId: await getCurrentTabId()},
+    func: PREVIEWER_resetFont,
+    args: [fontOptionIds],
+  });
 }
 
 /**
- * Called when the user selects a new font to preview.
+ * Resets the form.
  */
-function changeFont() {
+function resetForm() {
+  document.forms[0].reset();
+}
 
+/**
+ * Called when the user selects a new font to preview.
+ */
+async function changeFont() {
   var selector = document.getElementById('selector').value;
   var fontFamily = fontOptions.fontFamily || fonts[0];
   var subset = document.getElementById('fonts-subset').value;
   if (subset.indexOf('-ext') > -1) {
     subset = subset.split('-ext')[0] + ',' + subset;
   }
-  var fontUrl = '//fonts.googleapis.com/css?family=' + fontFamily;
+  var fontUrl = cssEndpoint + fontFamily;
```

Comment:
I believe removing the `subset` query parameter here means it will pull in font files for every subset that a font supports. Is that okay?

Reply:
I noticed the Google Fonts website doesnt append subset to the URL when filtering by language, and I see no difference in the font files loaded here:
https://fonts.googleapis.com/css?family=Nokora
https://fonts.googleapis.com/css?family=Nokora&subset=khmer
Even the ones in the docs show no difference:
https://developers.google.com/fonts/docs/getting_started#specifying_script_subsets
I've tweeted at google for more clarity-
https://twitter.com/pamelafox/status/1542540493761875968

Reply:
Interesting, I thought I saw it filtering by subset before. Anyway, that is good news.

Reply:
FYI, I found out from Google that subset is only used by super old browsers as now unicode-range is used instead. https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range
Given the subset query parameter is only applicable to 4% of browsers at this point, it's fine for the extension generated URLs to leave it off now.